DEPENDENCIES.rst

```diff
@@ -0,0 +1,139 @@
+###########
+Cyclus Core
```

Comment:
Probably not the best title, just 'Dependencies' works.

Also, this document is not linked to anywhere in the install instructions?